news/2018-08-17-the-qat-gazette-july-2018.md

```diff
@@ -0,0 +1,97 @@
+
+### New Beatmap Nominator Promotions
+
+Whilst our existing Beatmap Nominators were being reviewed, we were pleased to welcome a wave of some new and returning circle inspectors to the team. Please offer warm welcome and congratulations to the following on their admission into the Beatmap Nominators:
+
+[BOUYAAA](https://osu.ppy.sh/users/405449), [Cris-](https://osu.ppy.sh/users/6175280), [frogyfro](https://osu.ppy.sh/users/7010859), [Gottagof4st](https://osu.ppy.sh/users/6122935), [IceKalt](https://osu.ppy.sh/users/5410645), [MrSergio](https://osu.ppy.sh/users/2581696), [pimpG](https://osu.ppy.sh/users/328615), [riffy](https://osu.ppy.sh/users/597957), [Rizen](https://osu.ppy.sh/users/3056732), [Seto Kousuke](https://osu.ppy.sh/users/2857314), [UndeadCapulet](https://osu.ppy.sh/users/2523533), [Yauxo](https://osu.ppy.sh/users/898306) and [Yugu](https://osu.ppy.sh/users/3161834)!
+
+May their efforts bring you many Ranked-beatmap-sized bundles of happiness over the coming weeks, months, and maybe even years!
+
+### Beatmap Nominator Retirements
+
+Where some may be taking their first steps as a Nominator, others seek new pastures, new challenges, or perhaps simply a well-deserved rest. We thank the following departees for their tireless efforts and enduring vision:
+
+[Arzenvald](https://osu.ppy.sh/users/3027421), [Delis](https://osu.ppy.sh/users/1603923) and [Sorcerer](https://osu.ppy.sh/users/1913190).
+
+## Within the Quality Assurance Team
+
+by [Raiden](https://osu.ppy.sh/users/2239480)
+
+There have been numerous events that are of everyone's interest this month, which we’ll get into in a second — with regards to team structure, there were unfortunately no new additions to the Quality Assurance Team this month. Luckily no one has decided to leave us either though, which is always a positive!
+
+### Changes to Beatmap Spread Rules
+
+Beatmap spread rules have been quite stagnant over the last couple of years, although that has now finally changed! After a proposal made by returning Beatmap Nominator [UndeadCapulet](https://osu.ppy.sh/users/2523533) and four entire months of discussion amongst various community members, a consensus was finally reached. Thus, the new spread rules (which you can learn more about below) were amended into the Ranking Criteria, overwriting the old ones that we used to have.
+
+Most notable changes are applied to the *lowest possible difficulty of a set*, which now depends on the beatmap set's total drain time:
+
++ If it is less than **3:30 minutes**, the lowest difficulty must still be a Normal or lower.
++ If it is between **3:30 and 4:15 minutes**, the lowest difficulty can be a Hard.
++ If it is between **4:15 and 5:00 minutes**, it can be an Insane.
+
+Beatmaps that exceed 5:00 minutes in drain time are unaffected as they can still be ranked with just one difficulty.
```

Comment:
are -> remain (I wrote it like this dun reword that :(((( )

Reply:
sure thing